### ISSUE TITLE
Bootstrap v4 end of life

### ISSUE THREAD

**Issue by SpeedyD** [corowne/lorekeeper#887]
Currently, Lorekeeper as we know it is _still_ running on.. 4.1.3. (According to app.css that is.)

**Bootstrap v4 has recently reached end of life.**

It seems we have fairly little choice but to [migrate to 5.3.x](https://getbootstrap.com/docs/5.3/migration/). As there are a whopping 53 breaking matters, JQuery being dropped and support for older browsers ended, this is not a great solution.

I did find a Migration tool )https://github.com/coliff/bootstrap-5-migrate-tool) - but I have no idea how useful this is in the long run.

**Comment by itinerare** [comment]
Yeah it's on the vague to-do list/will happen at some point. It's not that bad a migration, really, though it does take a little bit.

**Comment by SpeedyD** [comment]
In combination with the laravelcollective/html deprecation, I can imagine some annoyances, though...

**Comment by itinerare** [comment]
I mean, they're not likely to happen simultaneously anyway, so... not that big a concern.

**Comment by SpeedyD** [comment]
Added link to https://palcarazm.github.io/bootstrap5-toggle/ in my original message, as reminder that we need to swap out the toggle when we get to this.

**Comment by liwoyadan** [comment]
Oh, I've actually done the update to v5 for an LK site in the past and it wasn't too bad...I'll go ahead and put this on my eventual to-do, and see about PRing it into develop sometime soon. I can probably knock in the FontAwesome v6 upgrade as well while I'm at it.

**Comment by itinerare** [comment]
Oh, that'd be wonderful! I'll need to split off a branch for v4 development (since it's a breaking change, functionally), but I can do that whenever you get a PR set up/handle the logistics there.
